app.js

```diff
@@ -0,0 +1,101 @@
+require("dotenv").config();
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+const md5 = require("md5");
+const User = require("./models/users");
+var jwt = require('jsonwebtoken');
+
+try {
+    mongoose.connect("mongodb://localhost:27017/users");
+    console.log("connected to db");
+} catch (error) {
+    handleError(error);
+}
+process.on('unhandledRejection', error => {
+    console.log('unhandledRejection', error.message);
+});
+const app = express();
+app.use(express.static("public"));
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.json());
+app.post("/user/register", (req, res) => {
+    try {
```

Comment:
Create a separate controller file which will contain functionality related to route.